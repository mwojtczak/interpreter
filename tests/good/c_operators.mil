function main () {
	a = 3;
	print(a);
	a += 3;
	print(a);
	a *= 2;
	print(a);
	a -= 2;
	print(a);
	a /= 2;
	print(a);
	a++;
	print(a);
	a--;
	print(a);
}