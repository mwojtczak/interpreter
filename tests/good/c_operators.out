3
6
12
10
5
6
5
